### DISCUSSION TITLE
Finding Maximum Textures [value,ori] = max(odf,'numLocal',3)

### DISCUSSION THREAD

**Discussion by snayir6** [mtex-toolbox/mtex#1023] [Ask Anything] [answered]
Hi, Mtex community,

I have a question about extraction local texture maximum point. I am using numLocal function to doing it but it only produces 1 local maximum even though there are 3, am I missing something would you help me to correct this?
Cheers,
Selda

```
odf = calcODF(ebsd('Ni-superalloy').orientations)
[value,ori] = max(odf,'numLocal',3)
```
value =

    9.4551

 
ori = orientation (show methods, plot)
  size: 1 x 1
  crystal symmetry : Ni-superalloy (m-3m)
  specimen symmetry: 1
 
  Bunge Euler angles in degree
     phi1     Phi    phi2    Inv.
  182.851 25.3123 164.274       0

**Comment by zmichels** [comment]
Hi,

I usually use the calcModes() function for ODFs.

https://mtex-toolbox.github.io/ODF.calcModes.html

Maybe try that and see if it works any better.

Zach

**Comment by kilir** [comment, marked as answer]
Hi, `max` uses `calcmodes` however consider you are looking at a polefigure and you compute the maximum from an odf. Given that you have cubic crystal symmetry, the result makes sense to have three 100 maxima from one odf maximum. If you like to try to find some very minor ones, you can try to decrease the resolution maybe, but I doubt that there'll be something.
Cheers, Rüdiger
